apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -719,3 +721,11 @@ export const isEntityPublic: ImpureGraphFunction<
   Promise<boolean>
 > = async (ctx, _, params) =>
   canViewEntity(ctx, { actorId: publicUserAccountId }, params);
+
+export const getEntityRelations: ImpureGraphFunction<
```

Comment:
I wonder whether it might be better to name this something less generic so that it's more clear that these "relations" are related to authentication? Maybe `getEntityAuthRelations`, or `getEntityAccessRelations`? cc @CiaranMn

Reply:
Yes, I would agree with making the name more specific. 

These might be better called `relationships` also... the relation is the target (e.g. a user), the relationship is the connection with a target and relation-type... maybe `getEntityAuthRelationships`? or `getEntityPermissionRelationships`?

Reply:
The method name is aligned with the terminology used in SpiceDB, where you have "relations" and "permissions". Relations are the thing you can specify, such as "A user is a _viewer_ of an entity", and permissions are derived from that, such as "A user _can view_ an entity". The term "Relationship" is used for the tuple containing the subject, the object, and either permissions _and_ relations, but this function only returns the relations and the subjects. 

Because of this, I wouldn't call it `getEntityPermissionRelationships`. `Relationships` itself could also be confusing I think. Also, I wouldn't use `Auth` because it could be either authentication or authorization.

Reply:
Discussed offline and we're going to use `getEntityAuthorizationRelationships`.